features/src/main/feature/feature.xml

```diff
@@ -76,6 +76,21 @@
         <bundle start-level="80">mvn:org.openhab/com.caucho.quercus/4.0.45</bundle>
     </feature>
 
+    <feature name="openhab-ui-habmin" description="HABmin" version="${project.version}">
+        <feature>openhab-runtime-base</feature>
+        <bundle start-level="80">mvn:org.openhab.ui/org.openhab.ui.habmin/${project.version}</bundle>
+    </feature>
+
+    <feature name="openhab-ui-habot" description="HABot" version="${project.version}">
+        <feature>openhab-runtime-base</feature>
```

Comment:
@wborn  / @maggu2810 should these maybe also depend on `openhab-ui-dashboard`?

Reply:
Yes!

Reply:
Added with https://github.com/openhab/openhab-webui/pull/16